src/parser.ts

```diff
@@ -63,7 +63,7 @@ function cleanupYaml(ref: any){
             let key = Object.keys(ref)[i];
 
             // Resolve the function
-            if(ref[key].hasOwnProperty('class') && ref[key].hasOwnProperty('data')){
+            if( ref[key] !== null && ref[key].hasOwnProperty('class') && ref[key].hasOwnProperty('data')){
```

Comment:
I found that I needed to make a couple of changes to the parser to block the script from trying to parse out null entries in yaml arrays.